app/controllers/application_controller.rb

```diff
@@ -0,0 +1,19 @@
+class ApplicationController < ActionController::Base
+  # Prevent CSRF attacks by raising an exception.
+  # For APIs, you may want to use :null_session instead.
+  protect_from_forgery with: :exception
+
+  helper_method :current_user
+
```

Comment:
Interesting use of the helper_method to make the current user available in the views.